src/pywinctl/_pywinctl_macos.py

```diff
@@ -86,39 +86,22 @@ def getActiveWindow(app: AppKit.NSApplication | None = None):
     :return: Window object or None
     """
     if not app:
-        # app = WS.frontmostApplication()   # This fails after using .activateWithOptions_()?!?!?!
-        cmd = """on run
-                    set appName to ""
-                    set appID to ""
-                    set winData to {}
-                    try
-                        tell application "System Events"
-                            set appName to name of first application process whose frontmost is true
-                            set appID to unix id of application process appName
-                            tell application process appName
-                                set winData to {position, size, name} of (first window whose value of attribute "AXMain" is true)
-                            end tell
-                        end tell
-                    end try
-                    return {appID, winData}
-                end run"""
-        proc = subprocess.Popen(['osascript'],  stdin=subprocess.PIPE, stdout=subprocess.PIPE, encoding='utf8')
-        ret, err = proc.communicate(cmd)
-        entries = ret.replace("\n", "").split(", ")
-        try:
-            appID = entries[0]
-            bounds = Rect(int(entries[1]), int(entries[2]), int(entries[3]), int(entries[4]))
```

Comment:
`entries[3]` is the width of the window (not the right bound)
`entries[4]` is the height of the window (not the bottom bound)

Reply:
Totally right! Furthermore, it has been wrong maybe eons ago! I'ts not noticed since the window rect is updated every time you query any of its parts... But the right way to define it is as you mention. Thank you again!